_posts/en/2021-06-08-working-in-the-open-part-2.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title:  "Working in the Open: Part 2"
+ref: working-in-the-open-p2
+lang: en
+author: "Guillaume Charest, in collaboration with the IT Strategy team"
+date: "2021-06-08"
+excerpt_separator: <!--more-->
+---
+This series of blog posts will explore the concept of "Working in the Open".
+In this second entry, we will attempt to explain how you can get started while respecting your duties as a public servant!
+<!--more-->
+
+In our [previous post]({{site.baseurl}}{% post_url 2019-11-19-working-in-the-open-part-1 %}), we explored what it meant to work in the open and why we should do so by default.
+This time, let's have a look at how we can accomplish this while ensuring we follow security and privacy best practices.
+
+_Note: The scope of this post will be more around everyday working in the open than software development.
+We may take a deeper dive in this other topic in an upcoming blog post._
+
+## Inner Source vs Open Source
+
+It's worth reminding ourselves that not everyone is mentally and emotionally ready to open up to the whole world.
+Various reasons exist and we should not judge.
+
+As mentioned in our [previous post]({{site.baseurl}}{% post_url 2019-11-19-working-in-the-open-part-1 %}), some organizations have taken the "Inner Source" road, opening up all workspaces to all their internal teams, to help folks slowly get comfortable with working in the open as well as reaping part of its benefits.
+Doing so can help identify what works well and what challenges surfaces for your team and your organization.
```

Comment:
I think the 's' should be removed from surfaces. So it would read '...what works well and what challenges surface'